### PR TITLE
test: improve e2e testing environment

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- Use the `CalendarExample` component to generate controlled calendar instances.
- Replace the `wait-for-expect` package with the `expect.«poll` API.
- Pin the rendering datetime for the view changing by date.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/playwright/month/gridSelection.e2e.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ test.describe('Selection', () => {
     const monthGridCellLocator = page.locator(MONTH_GRID_SELECTOR).nth(31);
 
     // When
-    await monthGridCellLocator.click({ delay: 1 });
+    await monthGridCellLocator.click({ delay: 100 });
```
Comment (jajugoguma) [inline apps/calendar/playwright/month/gridSelection.e2e.ts:42]: 딜레이 값을 상수처리하는 것도 좋을 것 같습니다!

File: apps/calendar/playwright/week/alldayGridEventDragAndDrop.e2e.ts [hunk 2]
```diff
@@ -87,16 +97,12 @@ test.describe('moving allday grid row event when moving by holding the middle or
...
-        boundingBoxBeforeMoving.x + (boundingBoxBeforeMoving.width * 2) / 3,
-        1
-      );
-      expect(boundingBoxAfterMoving.x).toBeLessThan(
-        boundingBoxBeforeMoving.x + boundingBoxBeforeMoving.width
-      );
-    });
+    await expect
+      .poll(() => getX(targetEventLocator))
+      .toBeCloseTo(boundingBoxBeforeMoving.x + (boundingBoxBeforeMoving.width * 2) / 3, 1);
```
Comment (jajugoguma) [inline apps/calendar/playwright/week/alldayGridEventDragAndDrop.e2e.ts:102]: `boundingBoxBeforeMoving`의 프로퍼티를 destructuring해서 사용해도 좋을 것 같습니다.

**Review by jajugoguma — APPROVED** [review]
리뷰 완료합니다!

File: apps/calendar/playwright/week/alldayGridEventDragAndDrop.e2e.ts (hunk 2 above)
Comment (adhrinae) [reply apps/calendar/playwright/week/alldayGridEventDragAndDrop.e2e.ts:102]: 요게 일단 무작정 분해해버리면 단언을 통해 무엇을 확인하려는지 보기 어려울 것 같아서 분해하지 않고 있어요.

File: apps/calendar/playwright/utils.ts [hunk 3]
```diff
@@ -65,13 +65,11 @@ export function getTimeStrFromDate(d: TZDate) {
   const hour = d.getHours();
   const minute = d.getMinutes();
 
-  return `${hour < 10 ? `0${hour}` : hour}:${minute < 10 ? `0${minute}` : minute}`;
+  return `${hour.toString().padStart(2, '0')}:${minute.toString().padStart(2, '0')}`;
```
Comment (lja1018) [inline apps/calendar/playwright/utils.ts:68]: `toString`을 위에 변수에서 바로 호출하면 어떨까요?

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다. 고생하셨습니다!

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다 :+1:

**Pull request merged**